dbt/seeds/etl_full_row_counts.csv

```diff
@@ -2618,14 +2649,14 @@ out_eia__yearly_assn_plant_parts_plant_gen,2014,374328
 out_eia__yearly_assn_plant_parts_plant_gen,2015,382015
 out_eia__yearly_assn_plant_parts_plant_gen,2016,397586
 out_eia__yearly_assn_plant_parts_plant_gen,2017,405480
-out_eia__yearly_assn_plant_parts_plant_gen,2018,421543
-out_eia__yearly_assn_plant_parts_plant_gen,2019,436448
-out_eia__yearly_assn_plant_parts_plant_gen,2020,450094
-out_eia__yearly_assn_plant_parts_plant_gen,2021,465425
-out_eia__yearly_assn_plant_parts_plant_gen,2022,479947
-out_eia__yearly_assn_plant_parts_plant_gen,2023,494220
-out_eia__yearly_assn_plant_parts_plant_gen,2024,492294
-out_eia__yearly_assn_plant_parts_plant_gen,2025,498246
+out_eia__yearly_assn_plant_parts_plant_gen,2018,421549
+out_eia__yearly_assn_plant_parts_plant_gen,2019,436454
+out_eia__yearly_assn_plant_parts_plant_gen,2020,450106
+out_eia__yearly_assn_plant_parts_plant_gen,2021,465441
+out_eia__yearly_assn_plant_parts_plant_gen,2022,479997
+out_eia__yearly_assn_plant_parts_plant_gen,2023,494226
+out_eia__yearly_assn_plant_parts_plant_gen,2024,509681
+out_eia__yearly_assn_plant_parts_plant_gen,2025,499520
```

Comment:
you think this is from the capitalization maybe? bc all the gens get fed into this